gui/src/components/maintenance/cortx-software.vue

```diff
@@ -136,16 +136,40 @@ export default class CortxHotfix extends Vue {
     isDirty: false,
     isValid: false,
   };
-
+  public systemStatus: boolean = true;
   public async mounted() {
+    await this.getSyetmStatus();
     await this.getLastUpgradeStatus();
   }
-
+ public async getSyetmStatus() {
+    this.$store.dispatch(
+      "systemConfig/showLoader",
+      "checking service status..."
+    );
+    try {
+      const res: any = await Api.getAll(apiRegister.system_status);
+      this.$store.dispatch("systemConfig/hideLoader");
+    } catch (error) {
+      this.$data.systemStatus = false;
+      let errorMessage = "please check service status";
```

Comment:
this text goes into en.json or respective json file

Reply:
done